Khan/wonder-blocks
# Update Banner docs in Storybook

## Summary:
After adding default args, the documentation got messed up.

Here, I added back the JS Docs on top of the Docs tab,
as well as the descriptions for the props in the panel.

**Storybook publish:**
https://5e1bf4b385e3fb0020b7073c-tnxhbhegrf.chromatic.com/?path=/docs/banner--default

Issue: https://khanacademy.atlassian.net/browse/WB-1385

<img width="1188" alt="Screen Shot 2022-10-25 at 4 50 07 PM" src="https://user-images.githubusercontent.com/13231763/197903116-872920f6-ed01-400c-b746-4ed3568bcd28.png">

<img width="1066" alt="Screen Shot 2022-10-25 at 4 54 43 PM" src="https://user-images.githubusercontent.com/13231763/197903241-9906b821-aa98-44bc-8135-246dabadc0cd.png">


## Test plan:
- Go to http://localhost:6061/?path=/docs/banner--default
- Check that the JS Docs and Usage appear on top.
- Read through the prop descriptions and confirm they're correct.
